drivers/video/sunxi/lcd_font.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <stdint.h>
+
+typedef struct {
+    const uint8_t* data;
+    uint8_t char_w;
+    uint8_t char_h;
+    uint8_t type;
+    uint8_t offset;
+} lcd_font_t;
+
+extern const lcd_font_t clock_digits;
+extern const lcd_font_t clock_dots;
+extern const lcd_font_t t_12x24_full;
+extern const lcd_font_t t_8x16_full;
```

Comment:
Any chance to add smaller fonts? Try basti79/LCDfonts , the 8x12 looks decent.

Reply:
Probably yes